### PR TITLE
[kube-prometheus-stack]: add labels spec for thanos-ruler

### PR DESCRIPTION
**Pull request by daniel-maganto**
#### What this PR does / why we need it:
This PR adds support for `labels` on spec for thanos-ruler template
#### Which issue this PR fixes
N/A

#### Checklist

- [X] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [X] Chart Version bumped
- [X] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

File: charts/kube-prometheus-stack/Chart.yaml [hunk 1]
```diff
@@ -23,7 +23,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 36.6.2
+version: 36.7.0
```
Comment (monotek) [inline charts/kube-prometheus-stack/Chart.yaml:26]:
```suggestion
version: 37.1.0
```

**Review by monotek — CHANGES_REQUESTED** [review]

File: charts/kube-prometheus-stack/Chart.yaml (hunk 1 above)
Comment (daniel-maganto) [reply charts/kube-prometheus-stack/Chart.yaml]: Fixed

**Comment by daniel-maganto** [comment]
I already have updated to the next minor version, right now 37.2.0

@monotek when you have time could you take a look. 

Many thanks

**Review by monotek — APPROVED** [review]

**Comment by monotek** [comment]
Can you please rebase?
I can't add changes to your repo.

**Pull request merged**